documentation/docs/09-amp.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: AMP
+title: Accelerated Mobile Pages (AMP)
 ---
 
 An unfortunate reality of modern web development is that it is sometimes necessary to create an [AMP](https://amp.dev/) version of your site. In SvelteKit this can be done by setting the [`amp`](/docs/configuration#amp) config option, which has the following effects:
```

Comment:
```suggestion
An unfortunate reality of modern web development is that it is sometimes necessary to create an [Accelerated Mobile Pages](https://amp.dev/) (AMP) version of your site. In SvelteKit this can be done by setting the [`amp`](/docs/configuration#amp) config option, which has the following effects:
```